### PR TITLE
Add save table endpoint

### PR DESCRIPTION
**Pull request by shashigharti**
- fixes #[118](https://github.com/frictionlessdata/application/issues/118)

---

Please make sure that all the checks pass. Please add here any additional information regarding this pull request. It's highly recommended that you link this PR to an issue (please create one if it doesn't exist for this PR)

### CODE REVIEW

File: frictionless/project/database.py [hunk 1]
```diff
@@ -281,3 +282,15 @@ def read_table(
         data = self.query(str(query))
         schema = record["resource"]["schema"]
         return ITable(tableSchema=schema, header=data["header"], rows=data["rows"])
+
+    def save_table(self, path: str, *, tablePatch: dict, basepath: str) -> Resource:
```
Comment (roll) [inline frictionless/project/database.py:291]: Please add TODO that's it's temporary while we don't have proper database's table editing

**Review by roll — COMMENTED** [review]
Thanks. Let's merge it as it is for now

**Comment by shashigharti** [comment]
Thank you @roll. It is ready for review!

**Review by roll — APPROVED** [review]
Thanks!

**Pull request merged**